README.md

```diff
@@ -592,3 +593,74 @@ Looking for a specific version of QM?
 Search in the mirrors list below.
 
 [CentOS Automotive SIG - qm package - noarch](https://mirror.stream.centos.org/SIGs/9-stream/automotive/aarch64/packages-main/Packages/q/)
+
+## Change configuration in QM
+
+To build the service that will run QM on ostree (which is an immutable OS) we use systemd units using Podman Quadlet.
+
+This means we cannot change the service file that holds the qm configuration and we need to add drop-in files to change the default configuration.
+
+To get more info about how podman-systemd.unit works please refer to the man page `man podman-systemd.unit`
+
+The default configuration drop-in file is the qm.container file and can be viewed in /usr/share/containers/systemd/qm.container.
+
+New drop-in files (ending in `.conf`) should be placed in /etc/containers/systemd/qm.container.d with new or changed configuration.
+
+For example, the default configuration for the `MemoryHigh` variable for QM is unlimited. Let's limit it to 2G.
+
+To check the current limit of QM memory we will run
+
+`systemctl status qm.service`
+
+and the output we get is
+
+`Memory: 25.8M (swap max: 0B peak: 29.8M)`
+
+which means there is no limit for QM to use `MemoryHigh`
+
+To make QM use 2G, we will need to change `MemoryHigh`.
+
+This variable is set to `infinity` and can be seen in `usr/share/containers/systemd/qm.container`.
+
+Let's create a drop-in file with the new configuration.
+
+To create a new configuration file, run:
+
+```bash
+mkdir -p /etc/containers/systemd/qm.container.d/
+touch /etc/containers/systemd/qm.container.d/100-MemoryMax.conf
+```
+
+and insert into that file this configuration
+
+```bash
+[Service]
+MemoryHigh=2G # MemoryHigh is specified in gigabytes (e.g., "2G" means 2 gigabytes)
+```
+
+To check if the new configuration that will be held for `qm.service` please run
```

Comment:
Verifying changes loaded and generated by by podman-system-generator
call podman-system-generator --dryrun